courses/models.py

```diff
@@ -440,6 +446,9 @@ def get_program_run_enrollments(cls, user, program):
         """
         return cls.objects.filter(user=user, run__course__program=program)
 
+    def to_dict(self):
+        return {**super().to_dict(), "readable_id": self.run.courseware_id}
```

Comment:
I know there might be other places in the codebase where `readable_id` might still be used for both models, but can you change the key here and at line 494 to `text_id`?